src/goto-instrument/dump_c.cpp

```diff
@@ -285,6 +285,13 @@ void dump_ct::operator()(std::ostream &os)
     os << std::endl;
   }
 
+  // global typedefs
+  for(const auto &td : typedef_map)
```

Comment:
This function (`operator()`) is already unmanageably large - could this perhaps be pulled out into a `dump_typedef_map(os) const` method?